src/Microsoft.HttpRepl.Tests/Commands/ChangeDirectoryCommandTests.cs

```diff
@@ -23,7 +24,9 @@ public void ExecuteAsync_UnknownEndpoint_DisplaysWarning()
 
             Setup(commandText: "cd NotAnEndpoint", out MockedShellState mockedShellState, out HttpState httpState, out ICoreParseResult parseResult);
 
-            httpState.Structure = new DirectoryStructure(null);
+            ApiDefinition apiDefinition = new ApiDefinition();
+            apiDefinition.DirectoryStructure = new DirectoryStructure(null);
```

Comment:
nit: use this pattern to initialize the object:
```
ApiDefinition apiDefinition = new ApiDefinition();
{
    DirectoryStructure = new DirectoryStructure(null),
};
```

Reply:
Discussed offline. Changed it here since my concern isn't relevant for these particular block of code.